R/goodnessoffitplot.R

```diff
@@ -51,14 +51,10 @@ GoodnessOfFitPlot.2Dreduction = function(object, max.points = 1000, ...) {
     if (object$normalized)
         x.title <- paste0(x.title, " (normalized)")
     title <- paste0(object$title, " - Shepard Diagram - Rank correlation: ", sprintf("%1.2f%%", 100 * correlation[2, 1]))
-    res <- Chart(y = y,
-        type = "Scatterplot",
-        title = title,
-        x.title = x.title,
-        y.title = "Output distance",
-        background.fill.opacity = 0,
-        charting.area.fill.opacity = 0,
-        legend.fill.opacity = 0)
+    res <- flipStandardCharts::CombinedScatter(x = y,
+                                               title = title,
```

Comment:
Why does it use `flipStandardCharts::CombinedScatter` instead of `rhtmlCombinedScatter::CombinedScatter`?

Reply:
I think you might as well change this to use `rhtmlCombinedScatter` as well. I noticed you replaced `Chart` in dimensionreductionscatterplot.R anyway. Goodness of Fit is always just a scatterplot - no bubbles or colors. Also, you've already removed `flipStandardCharts` from DESCRIPTION